GEOSldas_App/LDASsa_DEFAULT_inputs_ensupd.nml

```diff
@@ -2363,9 +2363,87 @@ obs_param_nml(53)%xcorr          = 0.
 obs_param_nml(53)%ycorr          = 0.
 obs_param_nml(53)%adapt          = 0
 
+! --------------------------------------------------------------------------------------------------------------
+!
+!  54 = CYGNSS_SM (CYGNSS Level 3 soil moisture version 3.2)
+!
+! https://podaac.jpl.nasa.gov/dataset/CYGNSS_L3_SOIL_MOISTURE_V3.2
+
+obs_param_nml(54)%descr          = 'CYGNSS_SM'
+obs_param_nml(54)%orbit          = 3
+obs_param_nml(54)%pol            = 0
+obs_param_nml(54)%N_ang          = 0
+obs_param_nml(54)%freq           = 0
+obs_param_nml(54)%FOV            = 20.
+obs_param_nml(54)%FOV_units      = 'km'
+obs_param_nml(54)%assim          = .false.
+obs_param_nml(54)%scale          = .false.
+obs_param_nml(54)%getinnov       = .false.
+obs_param_nml(54)%RTM_ID         = 0
+obs_param_nml(54)%bias_Npar      = 0
+obs_param_nml(54)%bias_trel      = 864000
+obs_param_nml(54)%bias_tcut      = 432000
+obs_param_nml(54)%nodata         = -9999.
+obs_param_nml(54)%varname        = 'sfmc'
+obs_param_nml(54)%units          = 'm3 m-3'
+obs_param_nml(54)%path           = '/discover/nobackup/amfox/CYGNSS_obs/'
+obs_param_nml(54)%name           = ''
+obs_param_nml(54)%maskpath       = ''
+obs_param_nml(54)%maskname       = ''
+obs_param_nml(54)%scalepath      = ''
+obs_param_nml(54)%scalename      = ''
+obs_param_nml(54)%flistpath      = ''
+obs_param_nml(54)%flistname      = ''
+obs_param_nml(54)%errstd         = 0.1
+obs_param_nml(54)%std_normal_max = 2.5
+obs_param_nml(54)%zeromean       = .true.
+obs_param_nml(54)%coarsen_pert   = .false.
```

Comment:
I recognize that coarsen_pert is set to false for SMAP Tb obs in the DEFAULT nml file, but in L4_SM ops it's set to true.  Maybe specify true here?  (I'd need to re-familiarize myself with what exactly happens if this is set to true.)